config_converter/config_converter.py

```diff
@@ -0,0 +1,215 @@
+"""Program that converts a fluentd config file to a master agent config file.
+
+Usage: Run the exec file in the bin folder with 2 arguments - the
+path to the config file to migrate, and a directory to store the output
+master agent config file in.
+
+To do:
+1. Improve the CLI. Include a --help option. Allow optional args like
+   log-level and log-filepath (see project docs).
+2. Create a mapping for log_level if there in the fluentd config file and input
+   default ones if not given
+3. Generate stats like #fields converted, ignored, and output as schema
+4. Generate logs
+5. Fix types of numbers, lists
+6. Generate tests to check if for right logs, right stats, above changes
+"""
+
+import copy
+import sys
+import os
+import subprocess
+import yaml
+from google.protobuf import json_format
+import config_converter.config_pb2 as config_pb2
```

Comment:
Could this be:

   from config_converter import config_pb2

?

Reply:
Oh that's neat - will change that!